docs/datagrid/understanding-dataGrid.md

```diff
@@ -0,0 +1,96 @@
+Understanding DataGrid
+
```

Comment:
Remove this line:
![image](https://github.com/highcharts/highcharts/assets/57004120/67221951-5616-4646-bf16-2f3c61ff7ec0)